catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -149,6 +160,23 @@
   private static final String SUCCESS_RESPONSE_TYPE = "success";
 
   private static final MetacardType SECURITY_ATTRIBUTES = new SecurityAttributes();
+
+  private static final QueryResponseImpl EMPTY_QUERY_RESPONSE =
+      new QueryResponseImpl(new QueryRequestImpl(null), Collections.emptyList(), 0);
+
+  private static final Executor INTRIGUE_EXECUTOR =
```

Comment:
✏️ it seems like these properties should be added to system.properties or at least documented somewhere

Reply:
added